### PR TITLE
Layer Panel: fix style issues

### PR DESCRIPTION
**Pull request by shaoshing**
This PR fix the following style issues:

#3558 Descenders are clipped a little in Layer names on Retina. Happens on some layer elements when `font-size` is `@base-8` (when `device-width <= 1440px`). 

screen shot 2016-01-19 at 5 29 40 pm
(the first line is not clipped, the sconed one is clipped slightly)

Another issue I found was that when element's height has decimal  (e.g. 25.1875px), their layout may be inconsistent in the screen. The picture below demonstrate the problem.

after
(when face's height is 25.1875px, the two folder icons has different relative position)

after-fixed
(If change height to 25px, everything looks fine.)

As a result, when double-click to edit title, some input will shift up by 1px.
layer-panel-offset

I cannot find an easy way to fix this, except for using the `@media` rule to tweak the face's rem height for a rounded pixel number in different screen resolution. @designedbyscience can you review and let me know if you will approve the solution? :hammer:

### CODE REVIEW

**Comment by shaoshing** [comment]
Another thing I noticed is that, Chrom is not using the correct font-size.

font-size
(`font-size: 50%` should equals to 8px, but it is 9px in the HTML element)

font-size-2
(Try change to 44%, but it is still using 9px)

**Comment by designedbyscience** [comment]
@shaoshing Still looking, but there is still some clipping on the highlighted layer name (on Retina, you can see it in your gif above actually)

**Comment by shaoshing** [comment]
@designedbyscience regarding text got clipped in the input, I noticied it yesterday and was doing some research.

First, I traced back to where we were using the flat structure, and found the exact same issue:

before - 9 6px
(input font-size is 9.6 px)
before - 9 6px editing
(in editing mode)

Then I tried different `font-size` in latest master, and found it happens when the input has the `font-size` between 9 and 10 plus when it is using the `Adobe Clean` type face.

##### `font-size: 8px` input text is correctly displayed
8px

##### `font-size: 9.6px` input text is clipped
9 6px

##### `font-size: 10px` input text is clipped
10px

##### `font-size: 11px` input text is correctly displayed
11px

##### `font-size: 9.6px` with default system font, input text is correctly displayed
9 6 px - regular font

Unfortunately I couldn't find any CSS attribute that controls the bottom space in text input (I tried `padding/box-sizing/line-height/height ...`). I am wondering if you have fought with this issue before, and do you have any suggestion on how we can resolve or bypass it?

File: src/style/sections/layers/face.less [hunk 1]
```diff
@@ -73,22 +73,19 @@
     flex-grow: 1;
     flex-shrink: 1;
     -webkit-user-select: none;
-    white-space: nowrap;
+    white-space: pre;
```
Comment (shaoshing) [inline src/style/sections/layers/face.less]: This is changed to `pre` to preserve multiple spaces in layer name.

**Comment by shaoshing** [comment]
Update: @designedbyscience I've added fix for the jump issue on non-retina screen.

**Comment by designedbyscience** [comment]
Woohoo! Looks good. 

a-b-yay

**Comment by placegraphichere** [comment]
this is great, thanks for diving in and fixing this @shaoshing

**Pull request merged**